easybuild/easyconfigs/b/BCFtools/BCFtools-1.8-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,46 @@
+easyblock = 'ConfigureMake'
+
+name = 'BCFtools'
+version = '1.8'
+
+homepage = 'http://www.htslib.org/'
+
+description = """
+ Samtools is a suite of programs for interacting with high-throughput
+ sequencing data.  BCFtools - Reading/writing BCF2/VCF/gVCF files and
+ calling/filtering/summarising SNP and short indel sequence variants
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/samtools/%(namelower)s/releases/download/%(version)s']
+sources = [SOURCELOWER_TAR_BZ2]
+checksums = ['4acbfd691f137742e0be63d09f516434f0faf617a5c60f466140e0677915fced']
+
+builddependencies = [
+    ('binutils', '2.28'),
+]
+
+dependencies = [
+    ('GSL', '2.4'),
+    ('HTSlib', '1.8'),
+    ('zlib', '1.2.11'),
+]
+
+configopts = """\
+    --enable-gsl\
+    --with-htslib=$EBROOTHTSLIB\
```

Comment:
@JackPerdue please make this a single line for consistency with existing easyconfigs, and to avoid making the configure command multi-line

Reply:
Fixed